tmt/utils.py

```diff
@@ -4424,6 +4424,35 @@ def normalize_shell_script_list(
     raise NormalizationError(key_address, value, 'a string or a list of strings')
 
 
+def normalize_shell_script(
+        key_address: str,
+        value: Union[None, str],
+        logger: tmt.log.Logger) -> Optional[ShellScript]:
+    """
+    Normalize a single string input value.
+
+    This is a fairly common input format present mostly in fmf nodes where
+    tmt, to make things easier for humans, allows this:
+
+    .. code-block:: yaml
+
+       foo: bar
+
+    Internally, we should stick to one type only, and make sure whatever we get
+    on the input, a string would be the output.
+
+    :param value: input value from key source.
+    """
```

Comment:
The docstring is probably suffering from the copy & paste. This would be fine, the rest is related to string-vs-list-of-strings situation:

```python
    """
    Normalize a single shell script input that may be unset.

    :param value: input value from key source.
    """
```

Reply:
@happz, thanks very much for review. The docstring is updated.